deployments/registry.go

```diff
@@ -28,6 +29,23 @@ const (
 	registryChartFile    = "container-registry-0.1.0.tgz"
 )
 
+var registryAuthMemo *auth.PasswordAuth
+
+func RegistryInstallAuth() (*auth.PasswordAuth, error) {
+	if registryAuthMemo == nil {
+		auth, err := auth.RandomPasswordAuth()
+		if err != nil {
+			return nil, err
+		}
+		registryAuthMemo = auth
+		// registryAuthMemo = &auth.PasswordAuth{
```

Comment:
Can we remove these comments now?

Reply:
oops. yes

Reply:
Done, 2ceb59f79c32d91a623818ebb927e5e855a6f709